### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by brunotm** [brunotm/replicant#5]
Missing documentation:

* Architecture
* Deployment
* Writing transaction drivers
* Writing tests for each driver

**Comment by Gisson** [comment]
@brunotm should this be divided into multiple issues? I also suggest to make a contributing guide with explicit guidelines.

**Comment by brunotm** [comment]
@Gisson agreed. What do you think on breaking it down in separate issues for the topics listed plus the contributing guide?

**Comment by Gisson** [comment]
@brunotm I think that should be the way to go yes.
